lib/rubocop/config_loader.rb

```diff
@@ -113,6 +113,11 @@ def configuration_from_file(config_file)
         config = load_file(config_file)
         return config if config_file == DEFAULT_FILE
 
+        if ConfigLoader.ignore_parent_exclusion
+          print 'Ignoring AllCops/Exclude from parent folders' if debug?
+          return merge_with_default(config, config_file)
```

Comment:
I would prefer an `else` here since we have a `merge_with_default(config, config_file)` at the end of the method that we can use.